decidim-core/app/assets/javascripts/decidim/form_filter.component.js.es6

```diff
@@ -1,12 +1,15 @@
 /* eslint-disable no-div-regex, no-useless-escape, no-param-reassign */
 
 /**
- * A component that handles the meetings filter form.
```

Comment:
I would clarify: "A plain JavaScript component" in order not to get confused with any other framework's components.